scss/_carousel.scss

```diff
@@ -193,3 +193,20 @@
   color: $carousel-caption-color;
   text-align: center;
 }
+
+// Dark mode carousel
+
+.carousel-dark {
+  .carousel-control-prev-icon,
+  .carousel-control-next-icon {
+    filter: invert(1) grayscale(100);
+  }
+
+  .carousel-indicators li {
+    background-color: $carousel-dark-indicator-active-bg;
+  }
+
+  .carousel-caption {
+    color: $carousel-dark-caption-color;
+  }
```

Comment:
There's a potential improvement here: both in current and dark versions, those values match.

Can we imagine setting `color` on `.carousel` directly, and:
1. use `currentColor`  for `.carousel-indicators li`'s `background-color`,
2. not declaring `.carousel-caption`'s `color` at all since it'd be inherited?

You'd have:
```scss
.carousel {
  color: $carousel-color;
}

.carousel-dark {
  color: $carousel-dark-color;
}
```

Reply:
Thought about that, but I think a more surgical approach here makes sense given the tendency for people to put other components inside one another. Setting it at the top level means if someone puts say a card or something else in a slide, it'll inherit that color.

Reply:
Aww you're right, I'm still not used to that kind of use cases.